src/GrampsJs.js

```diff
@@ -430,9 +429,6 @@ export class GrampsJs extends LitElement {
     if (this.loadingState === LOADING_STATE_MISSING_SETTINGS) {
       return this._renderOnboarding()
     }
-    if (this.settings.lang && Object.keys(this._strings).length === 0) {
-      this._loadStrings(grampsStrings, this.settings.lang)
-    }
```

Comment:
See comment below; that we are doing the call here makes sense as we know we are authorized at this point. However I do see the issue that `Object.keys(this._strings).length === 0` won't work as we have already loaded the frontend strings...